pages/report.tsx

```diff
@@ -75,15 +77,44 @@ function useInitialFormValues(): UnsubmittedFormResponse {
     email,
     should_subscribe: false,
     confirmed_get_app_updates: false,
+    image_uuid: formUuid,
   };
 }
 
 export default function Report() {
   const [currentStep, setCurrentStep] = useState(FormStep.SightingDetails);
   const { session } = useSessionUser();
-  const defaultFormFormValues = useInitialFormValues();
+  const [formUUID] = useState(uuidv4());
+  const defaultFormFormValues = useInitialFormValues(formUUID);
   const submitForm = useSubmitSharkSightingForm();
 
+  const splitFileReturnExtension = (fileName: string) => {
+    return fileName.split(".").pop();
+  };
+
+  const onFileChange = async (e: React.ChangeEvent<HTMLInputElement>) => {
+    const fileList = e.target.files;
+    if (!fileList) return;
+    for (let i = 0; i < fileList.length; i++) {
+      // 5242880 = 5mb
+      if (!fileList[i] || fileList[i].size > 5242880) return;
+      try {
+        await supabase.storage
+          .from("user-report-images")
+          .upload(
+            `./${formUUID}/${fileList[i].name}` +
+              splitFileReturnExtension(fileList[i].name),
```

Comment:
We don't need to do this anymore if we're keeping the original filename, right? Otherwise we're inserting names like `image.pngpng`